waspc/data/Generator/templates/sdk/server/operations/index.ts

```diff
@@ -1,4 +1,13 @@
+{{={= =}=}}
 // PUBLIC API
-export * from './queries/types'
+export * from './queries/types.js'
 // PUBLIC API
-export * from './actions/types'
+export * from './actions/types.js'
```

Comment:
This became a problem when I started using the file for something other than type imports.

Reply:
Maybe we bundle the SDK? Bundle everyting! 😂